db/migrate/20150105185344_add_admin_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAdminToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :admin, :boolean, default: false, null: false
```

Comment:
You dont need to make this change here, but in the future when you're namespacing, it's worth being flexible on what kinds of users there are. In that case, something like `role` would work better, where various roles could be monster-admin, baby-admin, user, mega-user, paying user, etc.